framework/validators/EachValidator.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\validators;
+
+use yii\base\InvalidConfigException;
+use Yii;
+use yii\base\Model;
+
+/**
+ * EachValidator serves validation of the array attributes.
+ * It perform validation of each array element using any other validator specified by [[rule]].
+ *
+ * ~~~php
+ * class MyModel extends Model
+ * {
+ *     public $arrayAttribute = [];
+ *
+ *     public function rules()
+ *     {
+ *         return [
+ *             ['arrayAttribute', 'each', 'rule' => ['trim']],
+ *             ['arrayAttribute', 'each', 'rule' => ['integer']],
+ *         ]
+ *     }
+ * }
+ * ~~~
+ *
+ * Note: this validator will not work with validation declared via model inline method.
```

Comment:
Why?
If you write your validation rule as model inline method, you may as well handle attribute array value inside it.
It does not make much sense to use such complex manipulation as `EachValidator` to actually do so.
Also, note, that `Closure` based validation will work just fine.